### ISSUE TITLE
Reading  from standard in broken?

### ISSUE THREAD

**Issue by francis36012** [shyiko/ktlint#51]
I'm trying to use the `--stdin` flag but ktlint doesn't seem to be getting anything from standard in.

Here is a transcript of my terminal session:

```shell
francis::aether:ktlint.git [master %=] cat test.kt 
import java.io.File

fun main(args: Array<String>) {
}
francis::aether:ktlint.git [master %=] ktlint --stdin --debug < test.kt
[DEBUG] Discovered ruleset "standard"
[DEBUG] Checking <text>
[DEBUG] 367ms / 0file(s) / 0error(s)
```

```
francis::aether:ktlint.git [master %=] ktlint --stdin --debug          
[DEBUG] Discovered ruleset "standard"
import java.io.File

fun main(args: Array<String>) {
}
[DEBUG] Checking <text>
[DEBUG] 11032ms / 0file(s) / 0error(s)
```

I was expecting an unused import error but as you can see above, no error is produced.

**Comment by shyiko** [comment]
Thank you for the ticket. Fixed in 0.8.1.
